src/humanpy/initialize.py

```diff
@@ -1,120 +1,141 @@
-
-        ikmodel_right = InverseKinematicsModel(
-                robot,
-                iktype=IkParameterizationType.Transform6D,
-                manip=robot.right_arm
-        )
-        if not ikmodel_right.load():
-            ikmodel_right.autogenerate()
-
-    #Setup planning pipeline
-    robot.planner = Sequence(
-            SnapPlanner(),
-            VectorFieldPlanner(),
-            CBiRRTPlanner(),
-    )
-    robot.simplifier = None
-    robot.retimer = HauserParabolicSmoother() # hack
-    robot.smoother = HauserParabolicSmoother()
-
-    robot.actions = prpy.action.ActionLibrary()
-
-    #Setup viewer. Default to load rviz
-    if attach_viewer == True:
-        attach_viewer = 'rviz'
-        env.SetViewer('attach_viewer')
-
-        #Fallback on qtcoin if rviz couldnt load
-        if env.GetViewer is None:
-            logger.warning('Loading the Rviz viewer failed. Falling back on qtcoin')
-            attach_viewer = 'qtcoin'
-
-    if attach_viewer and env.GetViewer() is None:
-        env.SetViewer(attach_viewer)
-        if env.GetViewer() is None:
-            raise Exception('Failed creating viewer of type "{0:s}"'.format(
-                attach_viewer))
-
-    #Remove ROS Logging since loading Rviz might have added it
-    prpy.logger.remove_ros_logger()
-
-    return env, robot
+PACKAGE = 'humanpy'
+import logging
+import os
+
+from catkin.find_in_workspaces import find_in_workspaces
+from openravepy import Environment, IkParameterizationType, RaveCreateController, RaveCreateModule
+from openravepy.databases.inversekinematics import InverseKinematicsModel
+
+from humanhand import HumanHand
+import humanpy.action # register actions
+import humanpy.tsr # register TSR libraries
+import prpy
+from prpy.base.endeffector import EndEffector
+from prpy.base.manipulator import Manipulator
+from prpy.base.robot import Robot
+from prpy.bind import bind_subclass
+from prpy.planning.base import Sequence
+from prpy.planning.cbirrt import CBiRRTPlanner
+from prpy.planning.vectorfield import VectorFieldPlanner
+from prpy.planning.snap import SnapPlanner 
+from prpy.planning.retimer import HauserParabolicSmoother
+import numpy
+
+logger = logging.getLogger('humanpy')
+
+def initialize(attach_viewer=False, sim=True, user_id='human', env=None):
+    """Initialize the Human Robot"""
+    
+    prpy.logger.initialize_logging()
+    
+    from catkin.find_in_workspaces import find_in_workspaces
+    package_name = 'humanpy'
+    directory = 'ordata/dae'
+    objects_path = find_in_workspaces(
+        search_dirs=['share'],
+        project=package_name,
+        path=directory,
+        first_match_only=True)
+    if len(objects_path) == 0:
+        print('Can\'t find directory %s/%s' % (package_name, directory))
+        sys.exit()
+    else:
+        print objects_path # for me this is '/home/spelle/storage/ros_test_ws/src/humanpy/ordata/dae/man1.dae'
+        objects_path = objects_path[0]

```

Comment:
This logic is not necessary to load `KinBody`s. Instead, you should use a [Catkin environment hook](http://docs.ros.org/fuerte/api/catkin/html/macros.html#catkin_add_env_hooks) to add `humanpy`'s `ordata` directory to the `OPENRAVE_PLUGINS` environment variable in the Catkin workspace's `setup.sh` file. See `pr_ordata`, `openrave_catkin`, or `or_fcl` for an example of this.